src/main/java/GridView.java

```diff
@@ -19,19 +18,19 @@ protected void paintComponent(Graphics g) {
 
     void paintGrid(Graphics g) {
         Square[][] board = grid.getBoard();
-        for (int y = 0; y < board.length; y++) {
-            for (int x = 0; x < board[y].length; x++) {
+        for (int y = 0; y < grid.HEIGHT; y++) {
+            for (int x = 0; x < grid.WIDTH; x++) {
```

Comment:
I want to say that using HEIGHT and WIDTH does not work. I changed it to board.length in the previously merged pr because otherwise it throws a null pointer

Reply:
I'm not sure why it doesn't work for you. It seems to run just fine on my local

Reply:
![image](https://user-images.githubusercontent.com/59262522/98574941-907ceb80-2286-11eb-9591-2dfcee176de2.png)

Reply:
I noticed in Grid that there wasn't an access modifier for HEIGHT and WIDTH. If it would be private, I wouldn't be able to access it without a getter, if it's protected, I can access it. We can decide which modifier to use, and whether we would like to create a getter.

Reply:
Interesting, if it works then that's great, but probably everything should be changed to be the same. It also could be that when I made lots of changes to the grid in the pr that hasn't yet been accepted for some reason using HEIGHT and WIDTH no longer work...but that's probs not the case.
Once Rachel accepts the grid pr I'll double check to make sure using HEIGHT and WIDTH work, and if they do then I'll switch everything over

Reply:
HEIGHT and WIDTH should probs be public? And then all you have to do is grid.HEIGHT or grid.WIDTH whenever you want them. No reason to make them private I dont think